src/game/Game.kt

```diff
@@ -2,31 +2,39 @@ package game
 
 import action.Action
 import agent.Player
+import map.AlwaysGoUpPlayer
 import map.ExitNode
 import map.WorldMap
 import java.lang.UnsupportedOperationException
 
 class Game(private val player: Player, private val worldMap: WorldMap) {
+    companion object {
```

Comment:
this feels more like enum. or exception type.

Reply:
Yea moving this to its own class called Exception Messages